python-lib/osisoft_plugin_common.py

```diff
@@ -65,6 +65,23 @@ def check_debug_mode(config):
     return config.get('show_advanced_parameters', False) and config.get('is_debug_mode', False)
 
 
+def check_must_convert_object_to_string(config):
+    return config.get('show_advanced_parameters', False) and config.get('must_convert_object_to_string', False)
```

Comment:
Q: why do we expectly test for `show_advabced_parameters` here (or in similar cases)? If we moved the parameter out of the advanced section this would stop working, and Im not sure what it adds to have that check.